src/services/api/repositories/petition.js

```diff
@@ -23,6 +23,11 @@ export default {
   update: (petition) => {
     const requestPath = path.join('/petitions', petition.id.toString());
     delete petition.id;
+
+    if (petition.city && !petition.city.data) {
+      delete petition.city.data;
+    }
```

Comment:
yeah this seems strange. please clarify with Lovely? they changed @pgoetze's request yesterday so that might be what broke it?